euclidean_gcd/C++/euclidean_gcd.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+using namespace std;
+
+int gcd(int a, int b){
+	if (a == 0)
+		return b;
+	return gcd(b % a, a); // returns gcd(remainder, a) if gcd(a,b) = r then gcd(a,b) = gcd(b,r)
+}
+int main(){
+	int a, b;
+	cout << "Enter a and b: " << endl;
+	cin >> a >> b;
+	cout << gcd(a,b);
+	return 0;
+}
```

Comment:
Could not find a newline character at the end of the file.  [whitespace/ending_newline] [5]

*Origin: CPPLintBear, Section: `all.cpplint`.*